client/backend/tfs-sdk/build.gradle

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root.
+
+plugins {
+    id "base"
+    id "de.undercouch.download"
+}
+
+ext {
+    sdkVersion = "14.135.0"
+    sdkArchiveLocation = file("$buildDir/TFS-SDK-${sdkVersion}.zip")
+    sdkUnpackLocation = file("$buildDir/sdk")
+    sdkJarName = "com.microsoft.tfs.sdk-${sdkVersion}.jar"
+}
+
+task downloadArchive(type: Download) {
+    src "https://github.com/JetBrains/team-explorer-everywhere/releases/download/$sdkVersion/TFS-SDK-${sdkVersion}.zip"
+    dest sdkArchiveLocation
+    overwrite false
```

Comment:
No. If the download is interrupted, then the `verifyArchive` task will catch that (because the hashes will disagree). After that, the user should call `./gradlew :client:backend:tfs-sdk:clean` manually and start over.